sql/src/main/java/org/apache/druid/sql/calcite/planner/IngestHandler.java

```diff
@@ -280,6 +281,7 @@ public String statementKind()
       return DruidSqlInsert.OPERATOR.getName();
     }
 
+    @Nullable
```

Comment:
I dont think this needs to be on this particular implementation since it is not nullable. Or does it complain if it doesnt match the interface annotations?

Reply:
I think some IDEs and tools don't inherit the annotations from the interface to spot bugs, so the recommendation is to add in both places, even though it's redundant